src/services/web3/governance/governance.ts

```diff
@@ -31,13 +31,13 @@ export const stakeAmount = async (
     const expandedAmount = expandToken(amount, govToken.decimals);
 
     const networkVars = await networkVars$.pipe(take(1)).toPromise();
-    const govContract = buildGovernanceContract(
+    const govContract = Governance__factory.connect(
       networkVars.governanceContractAddress,
       writeWeb3
     );
 
     const txHash = await resolveTxOnConfirmation({
-      tx: govContract.methods.stake(expandedAmount),
+      tx: await govContract.stake(expandedAmount),
```

Comment:
just a question - you added `await` - was it async before? I didn't notice that change to using the tx field and I want to make sure that this is not a redundant addition or that a necessary change to the usage was overlooked